R/modify_pd.R

```diff
@@ -8,33 +8,93 @@ NULL
 
 #' @rdname update_indention
 indent_round <- function(pd, indent_by) {
-  opening <- which(pd$token == "'('")
-  if (length(opening) > 0) {
+  indention_needed <- needs_indention(pd, token = "'('")
+  if (indention_needed) {
+    opening <- which(pd$token == "'('")
     start <- opening + 1
     stop <- nrow(pd) - 1
   } else {
     start <- stop <- 0
```

Comment:
Yeah I think that's true. I was referring to that under To Do in the PR description.